2023-01-26 00:32:18,193 WARNING  log log.py Not Found: /media/photos/2023/01/18/15910177301560.png 
2023-01-26 00:32:18,196 WARNING  log log.py Not Found: /media/photos/2023/01/15/scale_1200.webp 
2023-01-26 00:32:18,199 WARNING  log log.py Not Found: /media/photos/2023/01/02/543-9a6d5dc5b946eb9a62d2f62422af011c.png 
2023-01-26 00:32:18,200 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/18/15910177301560.png HTTP/1.1" 404 17 
2023-01-26 00:32:18,202 WARNING  log log.py Not Found: /media/photos/2023/01/10/7660079.png 
2023-01-26 00:32:18,204 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/15/scale_1200.webp HTTP/1.1" 404 17 
2023-01-26 00:32:18,206 WARNING  log log.py Not Found: /media/photos/2023/01/07/15800683022690.jpg 
2023-01-26 00:32:18,207 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/543-9a6d5dc5b946eb9a62d2f62422af011c.png HTTP/1.1" 404 17 
2023-01-26 00:32:18,209 WARNING  log log.py Not Found: /media/photos/2023/01/05/Lukashenko_potato_big.jpg 
2023-01-26 00:32:18,211 WARNING  log log.py Not Found: /media/photos/2023/01/06/You_Shall_Not_Pass_0-1_screenshot.jpg 
2023-01-26 00:32:18,212 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/07/15800683022690.jpg HTTP/1.1" 404 17 
2023-01-26 00:32:18,214 WARNING  log log.py Not Found: /media/photos/2023/01/03/iStock-579733578-e1573574017848-1024x499.jpg 
2023-01-26 00:32:18,215 WARNING  log log.py Not Found: /media/photos/2023/01/02/scale_1200.webp 
2023-01-26 00:32:18,216 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/10/7660079.png HTTP/1.1" 404 17 
2023-01-26 00:32:18,217 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/05/Lukashenko_potato_big.jpg HTTP/1.1" 404 17 
2023-01-26 00:32:18,218 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/06/You_Shall_Not_Pass_0-1_screenshot.jpg HTTP/1.1" 404 17 
2023-01-26 00:32:18,218 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/03/iStock-579733578-e1573574017848-1024x499.jpg HTTP/1.1" 404 17 
2023-01-26 00:32:18,220 WARNING  log log.py Not Found: /media/photos/2022/12/28/84482761-e1af-4749-ac54-ac2d26e2e8fb.jpg 
2023-01-26 00:32:18,222 WARNING  log log.py Not Found: /media/photos/2023/01/02/slug-3612269_1.jpg 
2023-01-26 00:32:18,223 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/scale_1200.webp HTTP/1.1" 404 17 
2023-01-26 00:32:18,225 WARNING  basehttp basehttp.py "GET /media/photos/2022/12/28/84482761-e1af-4749-ac54-ac2d26e2e8fb.jpg HTTP/1.1" 404 17 
2023-01-26 00:32:18,225 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/slug-3612269_1.jpg HTTP/1.1" 404 17 
2023-01-26 00:32:20,271 WARNING  log log.py Not Found: /media/photos/2023/01/18/15910177301560.png 
2023-01-26 00:32:20,273 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/18/15910177301560.png HTTP/1.1" 404 17 
2023-01-26 00:32:20,279 WARNING  log log.py Not Found: /media/photos/2023/01/15/scale_1200.webp 
2023-01-26 00:32:20,280 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/15/scale_1200.webp HTTP/1.1" 404 17 
2023-01-26 00:32:20,325 WARNING  log log.py Not Found: /media/photos/2023/01/02/543-9a6d5dc5b946eb9a62d2f62422af011c.png 
2023-01-26 00:32:20,327 WARNING  log log.py Not Found: /media/photos/2023/01/10/7660079.png 
2023-01-26 00:32:20,329 WARNING  log log.py Not Found: /media/photos/2023/01/07/15800683022690.jpg 
2023-01-26 00:32:20,331 WARNING  log log.py Not Found: /media/photos/2023/01/05/Lukashenko_potato_big.jpg 
2023-01-26 00:32:20,332 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/543-9a6d5dc5b946eb9a62d2f62422af011c.png HTTP/1.1" 404 17 
2023-01-26 00:32:20,333 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/10/7660079.png HTTP/1.1" 404 17 
2023-01-26 00:32:20,333 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/07/15800683022690.jpg HTTP/1.1" 404 17 
2023-01-26 00:32:20,333 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/05/Lukashenko_potato_big.jpg HTTP/1.1" 404 17 
2023-01-26 00:32:20,353 WARNING  log log.py Not Found: /media/photos/2023/01/06/You_Shall_Not_Pass_0-1_screenshot.jpg 
2023-01-26 00:32:20,355 WARNING  log log.py Not Found: /media/photos/2023/01/03/iStock-579733578-e1573574017848-1024x499.jpg 
2023-01-26 00:32:20,356 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/03/iStock-579733578-e1573574017848-1024x499.jpg HTTP/1.1" 404 17 
2023-01-26 00:32:20,356 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/06/You_Shall_Not_Pass_0-1_screenshot.jpg HTTP/1.1" 404 17 
2023-01-26 00:32:20,361 WARNING  log log.py Not Found: /media/photos/2023/01/02/scale_1200.webp 
2023-01-26 00:32:20,363 WARNING  log log.py Not Found: /media/photos/2022/12/28/84482761-e1af-4749-ac54-ac2d26e2e8fb.jpg 
2023-01-26 00:32:20,364 WARNING  log log.py Not Found: /media/photos/2023/01/02/slug-3612269_1.jpg 
2023-01-26 00:32:20,365 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/scale_1200.webp HTTP/1.1" 404 17 
2023-01-26 00:32:20,365 WARNING  basehttp basehttp.py "GET /media/photos/2022/12/28/84482761-e1af-4749-ac54-ac2d26e2e8fb.jpg HTTP/1.1" 404 17 
2023-01-26 00:32:20,366 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/slug-3612269_1.jpg HTTP/1.1" 404 17 
2023-01-26 00:32:22,811 WARNING  log log.py Not Found: /media/photos/2023/01/18/15910177301560.png 
2023-01-26 00:32:22,813 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/18/15910177301560.png HTTP/1.1" 404 17 
2023-01-26 00:32:22,818 WARNING  log log.py Not Found: /media/photos/2023/01/15/scale_1200.webp 
2023-01-26 00:32:22,820 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/15/scale_1200.webp HTTP/1.1" 404 17 
2023-01-26 00:32:22,865 WARNING  log log.py Not Found: /media/photos/2023/01/02/543-9a6d5dc5b946eb9a62d2f62422af011c.png 
2023-01-26 00:32:22,866 WARNING  log log.py Not Found: /media/photos/2023/01/10/7660079.png 
2023-01-26 00:32:22,868 WARNING  log log.py Not Found: /media/photos/2023/01/07/15800683022690.jpg 
2023-01-26 00:32:22,870 WARNING  log log.py Not Found: /media/photos/2023/01/05/Lukashenko_potato_big.jpg 
2023-01-26 00:32:22,871 WARNING  log log.py Not Found: /media/photos/2023/01/06/You_Shall_Not_Pass_0-1_screenshot.jpg 
2023-01-26 00:32:22,873 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/543-9a6d5dc5b946eb9a62d2f62422af011c.png HTTP/1.1" 404 17 
2023-01-26 00:32:22,873 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/10/7660079.png HTTP/1.1" 404 17 
2023-01-26 00:32:22,873 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/05/Lukashenko_potato_big.jpg HTTP/1.1" 404 17 
2023-01-26 00:32:22,874 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/07/15800683022690.jpg HTTP/1.1" 404 17 
2023-01-26 00:32:22,874 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/06/You_Shall_Not_Pass_0-1_screenshot.jpg HTTP/1.1" 404 17 
2023-01-26 00:32:22,892 WARNING  log log.py Not Found: /media/photos/2023/01/03/iStock-579733578-e1573574017848-1024x499.jpg 
2023-01-26 00:32:22,893 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/03/iStock-579733578-e1573574017848-1024x499.jpg HTTP/1.1" 404 17 
2023-01-26 00:32:22,904 WARNING  log log.py Not Found: /media/photos/2023/01/02/scale_1200.webp 
2023-01-26 00:32:22,906 WARNING  log log.py Not Found: /media/photos/2022/12/28/84482761-e1af-4749-ac54-ac2d26e2e8fb.jpg 
2023-01-26 00:32:22,906 WARNING  log log.py Not Found: /media/photos/2023/01/02/slug-3612269_1.jpg 
2023-01-26 00:32:22,907 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/scale_1200.webp HTTP/1.1" 404 17 
2023-01-26 00:32:22,908 WARNING  basehttp basehttp.py "GET /media/photos/2022/12/28/84482761-e1af-4749-ac54-ac2d26e2e8fb.jpg HTTP/1.1" 404 17 
2023-01-26 00:32:22,909 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/slug-3612269_1.jpg HTTP/1.1" 404 17 
2023-01-26 00:32:44,307 WARNING  log log.py Not Found: /media/photos/2023/01/18/15910177301560.png 
2023-01-26 00:32:44,309 WARNING  log log.py Not Found: /media/photos/2023/01/15/scale_1200.webp 
2023-01-26 00:32:44,311 WARNING  log log.py Not Found: /media/photos/2023/01/02/543-9a6d5dc5b946eb9a62d2f62422af011c.png 
2023-01-26 00:32:44,312 WARNING  log log.py Not Found: /media/photos/2023/01/10/7660079.png 
2023-01-26 00:32:44,313 WARNING  log log.py Not Found: /media/photos/2023/01/07/15800683022690.jpg 
2023-01-26 00:32:44,315 WARNING  log log.py Not Found: /media/photos/2023/01/05/Lukashenko_potato_big.jpg 
2023-01-26 00:32:44,316 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/18/15910177301560.png HTTP/1.1" 404 17 
2023-01-26 00:32:44,316 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/15/scale_1200.webp HTTP/1.1" 404 17 
2023-01-26 00:32:44,317 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/10/7660079.png HTTP/1.1" 404 17 
2023-01-26 00:32:44,317 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/543-9a6d5dc5b946eb9a62d2f62422af011c.png HTTP/1.1" 404 17 
2023-01-26 00:32:44,318 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/07/15800683022690.jpg HTTP/1.1" 404 17 
2023-01-26 00:32:44,318 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/05/Lukashenko_potato_big.jpg HTTP/1.1" 404 17 
2023-01-26 00:32:44,341 WARNING  log log.py Not Found: /media/photos/2023/01/06/You_Shall_Not_Pass_0-1_screenshot.jpg 
2023-01-26 00:32:44,343 WARNING  log log.py Not Found: /media/photos/2023/01/03/iStock-579733578-e1573574017848-1024x499.jpg 
2023-01-26 00:32:44,345 WARNING  log log.py Not Found: /media/photos/2023/01/02/scale_1200.webp 
2023-01-26 00:32:44,347 WARNING  log log.py Not Found: /media/photos/2022/12/28/84482761-e1af-4749-ac54-ac2d26e2e8fb.jpg 
2023-01-26 00:32:44,349 WARNING  log log.py Not Found: /media/photos/2023/01/02/slug-3612269_1.jpg 
2023-01-26 00:32:44,351 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/03/iStock-579733578-e1573574017848-1024x499.jpg HTTP/1.1" 404 17 
2023-01-26 00:32:44,351 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/06/You_Shall_Not_Pass_0-1_screenshot.jpg HTTP/1.1" 404 17 
2023-01-26 00:32:44,351 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/scale_1200.webp HTTP/1.1" 404 17 
2023-01-26 00:32:44,352 WARNING  basehttp basehttp.py "GET /media/photos/2022/12/28/84482761-e1af-4749-ac54-ac2d26e2e8fb.jpg HTTP/1.1" 404 17 
2023-01-26 00:32:44,353 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/slug-3612269_1.jpg HTTP/1.1" 404 17 
2023-01-26 15:24:42,616 WARNING  log log.py Not Found: /media/photos/2023/01/18/15910177301560.png 
2023-01-26 15:24:42,618 WARNING  log log.py Not Found: /media/photos/2023/01/15/scale_1200.webp 
2023-01-26 15:24:42,621 WARNING  log log.py Not Found: /media/photos/2023/01/09/16293175060633.jpg 
2023-01-26 15:24:42,623 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/18/15910177301560.png HTTP/1.1" 404 17 
2023-01-26 15:24:42,623 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/15/scale_1200.webp HTTP/1.1" 404 17 
2023-01-26 15:24:42,626 WARNING  log log.py Not Found: /media/photos/2023/01/02/543-9a6d5dc5b946eb9a62d2f62422af011c.png 
2023-01-26 15:24:42,633 WARNING  log log.py Not Found: /media/photos/2023/01/10/7660079.png 
2023-01-26 15:24:42,636 WARNING  log log.py Not Found: /media/photos/2023/01/05/Lukashenko_potato_big.jpg 
2023-01-26 15:24:42,638 WARNING  log log.py Not Found: /media/photos/2023/01/06/You_Shall_Not_Pass_0-1_screenshot.jpg 
2023-01-26 15:24:42,639 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/09/16293175060633.jpg HTTP/1.1" 404 17 
2023-01-26 15:24:42,639 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/543-9a6d5dc5b946eb9a62d2f62422af011c.png HTTP/1.1" 404 17 
2023-01-26 15:24:42,641 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/10/7660079.png HTTP/1.1" 404 17 
2023-01-26 15:24:42,644 WARNING  log log.py Not Found: /media/photos/2023/01/07/15800683022690.jpg 
2023-01-26 15:24:42,644 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/05/Lukashenko_potato_big.jpg HTTP/1.1" 404 17 
2023-01-26 15:24:42,645 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/06/You_Shall_Not_Pass_0-1_screenshot.jpg HTTP/1.1" 404 17 
2023-01-26 15:24:42,648 WARNING  log log.py Not Found: /media/photos/2023/01/03/iStock-579733578-e1573574017848-1024x499.jpg 
2023-01-26 15:24:42,655 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/07/15800683022690.jpg HTTP/1.1" 404 17 
2023-01-26 15:24:42,658 WARNING  log log.py Not Found: /media/photos/2022/12/28/84482761-e1af-4749-ac54-ac2d26e2e8fb.jpg 
2023-01-26 15:24:42,660 WARNING  log log.py Not Found: /media/photos/2023/01/02/slug-3612269_1.jpg 
2023-01-26 15:24:42,663 WARNING  log log.py Not Found: /media/photos/2023/01/02/scale_1200.webp 
2023-01-26 15:24:42,666 WARNING  basehttp basehttp.py "GET /media/photos/2022/12/28/84482761-e1af-4749-ac54-ac2d26e2e8fb.jpg HTTP/1.1" 404 17 
2023-01-26 15:24:42,667 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/03/iStock-579733578-e1573574017848-1024x499.jpg HTTP/1.1" 404 17 
2023-01-26 15:24:42,668 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/slug-3612269_1.jpg HTTP/1.1" 404 17 
2023-01-26 15:24:42,669 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/scale_1200.webp HTTP/1.1" 404 17 
2023-01-26 15:24:44,982 WARNING  log log.py Not Found: /media/photos/2023/01/18/15910177301560.png 
2023-01-26 15:24:44,983 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/18/15910177301560.png HTTP/1.1" 404 17 
2023-01-26 15:35:08,228 WARNING  log log.py Not Found: /media/photos/2023/01/18/15910177301560.png 
2023-01-26 15:35:08,230 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/18/15910177301560.png HTTP/1.1" 404 17 
2023-01-26 15:35:12,271 WARNING  log log.py Not Found: /media/photos/2023/01/18/15910177301560.png 
2023-01-26 15:35:12,272 WARNING  log log.py Not Found: /media/photos/2023/01/15/scale_1200.webp 
2023-01-26 15:35:12,276 WARNING  log log.py Not Found: /media/photos/2023/01/09/16293175060633.jpg 
2023-01-26 15:35:12,278 WARNING  log log.py Not Found: /media/photos/2023/01/02/543-9a6d5dc5b946eb9a62d2f62422af011c.png 
2023-01-26 15:35:12,278 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/18/15910177301560.png HTTP/1.1" 404 17 
2023-01-26 15:35:12,279 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/15/scale_1200.webp HTTP/1.1" 404 17 
2023-01-26 15:35:12,282 WARNING  log log.py Not Found: /media/photos/2023/01/10/7660079.png 
2023-01-26 15:35:12,282 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/09/16293175060633.jpg HTTP/1.1" 404 17 
2023-01-26 15:35:12,286 WARNING  log log.py Not Found: /media/photos/2023/01/07/15800683022690.jpg 
2023-01-26 15:35:12,286 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/543-9a6d5dc5b946eb9a62d2f62422af011c.png HTTP/1.1" 404 17 
2023-01-26 15:35:12,288 WARNING  log log.py Not Found: /media/photos/2023/01/06/You_Shall_Not_Pass_0-1_screenshot.jpg 
2023-01-26 15:35:12,290 WARNING  log log.py Not Found: /media/photos/2023/01/03/iStock-579733578-e1573574017848-1024x499.jpg 
2023-01-26 15:35:12,292 WARNING  log log.py Not Found: /media/photos/2023/01/05/Lukashenko_potato_big.jpg 
2023-01-26 15:35:12,293 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/10/7660079.png HTTP/1.1" 404 17 
2023-01-26 15:35:12,294 WARNING  log log.py Not Found: /media/photos/2023/01/02/scale_1200.webp 
2023-01-26 15:35:12,295 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/07/15800683022690.jpg HTTP/1.1" 404 17 
2023-01-26 15:35:12,296 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/03/iStock-579733578-e1573574017848-1024x499.jpg HTTP/1.1" 404 17 
2023-01-26 15:35:12,296 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/06/You_Shall_Not_Pass_0-1_screenshot.jpg HTTP/1.1" 404 17 
2023-01-26 15:35:12,297 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/05/Lukashenko_potato_big.jpg HTTP/1.1" 404 17 
2023-01-26 15:35:12,299 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/scale_1200.webp HTTP/1.1" 404 17 
2023-01-26 15:35:12,305 WARNING  log log.py Not Found: /media/photos/2022/12/28/84482761-e1af-4749-ac54-ac2d26e2e8fb.jpg 
2023-01-26 15:35:12,307 WARNING  log log.py Not Found: /media/photos/2023/01/02/slug-3612269_1.jpg 
2023-01-26 15:35:12,308 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/slug-3612269_1.jpg HTTP/1.1" 404 17 
2023-01-26 15:35:12,308 WARNING  basehttp basehttp.py "GET /media/photos/2022/12/28/84482761-e1af-4749-ac54-ac2d26e2e8fb.jpg HTTP/1.1" 404 17 
2023-01-26 15:35:52,597 WARNING  log log.py Not Found: /media/photos/2023/01/18/15910177301560.png 
2023-01-26 15:35:52,600 WARNING  log log.py Not Found: /media/photos/2023/01/15/scale_1200.webp 
2023-01-26 15:35:52,601 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/18/15910177301560.png HTTP/1.1" 404 17 
2023-01-26 15:35:52,603 WARNING  log log.py Not Found: /media/photos/2023/01/09/16293175060633.jpg 
2023-01-26 15:35:52,606 WARNING  log log.py Not Found: /media/photos/2023/01/02/543-9a6d5dc5b946eb9a62d2f62422af011c.png 
2023-01-26 15:35:52,607 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/15/scale_1200.webp HTTP/1.1" 404 17 
2023-01-26 15:35:52,609 WARNING  log log.py Not Found: /media/photos/2023/01/10/7660079.png 
2023-01-26 15:35:52,610 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/09/16293175060633.jpg HTTP/1.1" 404 17 
2023-01-26 15:35:52,612 WARNING  log log.py Not Found: /media/photos/2023/01/05/Lukashenko_potato_big.jpg 
2023-01-26 15:35:52,614 WARNING  log log.py Not Found: /media/photos/2023/01/06/You_Shall_Not_Pass_0-1_screenshot.jpg 
2023-01-26 15:35:52,615 WARNING  log log.py Not Found: /media/photos/2023/01/07/15800683022690.jpg 
2023-01-26 15:35:52,617 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/10/7660079.png HTTP/1.1" 404 17 
2023-01-26 15:35:52,617 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/543-9a6d5dc5b946eb9a62d2f62422af011c.png HTTP/1.1" 404 17 
2023-01-26 15:35:52,619 WARNING  log log.py Not Found: /media/photos/2023/01/03/iStock-579733578-e1573574017848-1024x499.jpg 
2023-01-26 15:35:52,620 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/05/Lukashenko_potato_big.jpg HTTP/1.1" 404 17 
2023-01-26 15:35:52,620 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/06/You_Shall_Not_Pass_0-1_screenshot.jpg HTTP/1.1" 404 17 
2023-01-26 15:35:52,621 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/07/15800683022690.jpg HTTP/1.1" 404 17 
2023-01-26 15:35:52,623 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/03/iStock-579733578-e1573574017848-1024x499.jpg HTTP/1.1" 404 17 
2023-01-26 15:35:52,625 WARNING  log log.py Not Found: /media/photos/2023/01/02/scale_1200.webp 
2023-01-26 15:35:52,627 WARNING  log log.py Not Found: /media/photos/2022/12/28/84482761-e1af-4749-ac54-ac2d26e2e8fb.jpg 
2023-01-26 15:35:52,631 WARNING  log log.py Not Found: /media/photos/2023/01/02/slug-3612269_1.jpg 
2023-01-26 15:35:52,632 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/scale_1200.webp HTTP/1.1" 404 17 
2023-01-26 15:35:52,632 WARNING  basehttp basehttp.py "GET /media/photos/2022/12/28/84482761-e1af-4749-ac54-ac2d26e2e8fb.jpg HTTP/1.1" 404 17 
2023-01-26 15:35:52,633 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/slug-3612269_1.jpg HTTP/1.1" 404 17 
2023-01-26 15:35:55,957 WARNING  log log.py Not Found: /media/photos/2023/01/18/15910177301560.png 
2023-01-26 15:35:55,959 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/18/15910177301560.png HTTP/1.1" 404 17 
2023-01-26 15:35:55,980 WARNING  log log.py Not Found: /media/photos/2023/01/15/scale_1200.webp 
2023-01-26 15:35:55,982 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/15/scale_1200.webp HTTP/1.1" 404 17 
2023-01-26 15:35:56,008 WARNING  log log.py Not Found: /media/photos/2023/01/09/16293175060633.jpg 
2023-01-26 15:35:56,011 WARNING  log log.py Not Found: /media/photos/2023/01/02/543-9a6d5dc5b946eb9a62d2f62422af011c.png 
2023-01-26 15:35:56,011 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/09/16293175060633.jpg HTTP/1.1" 404 17 
2023-01-26 15:35:56,013 WARNING  log log.py Not Found: /media/photos/2023/01/10/7660079.png 
2023-01-26 15:35:56,014 WARNING  log log.py Not Found: /media/photos/2023/01/07/15800683022690.jpg 
2023-01-26 15:35:56,015 WARNING  log log.py Not Found: /media/photos/2023/01/05/Lukashenko_potato_big.jpg 
2023-01-26 15:35:56,017 WARNING  log log.py Not Found: /media/photos/2023/01/06/You_Shall_Not_Pass_0-1_screenshot.jpg 
2023-01-26 15:35:56,019 WARNING  log log.py Not Found: /media/photos/2023/01/03/iStock-579733578-e1573574017848-1024x499.jpg 
2023-01-26 15:35:56,019 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/543-9a6d5dc5b946eb9a62d2f62422af011c.png HTTP/1.1" 404 17 
2023-01-26 15:35:56,020 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/10/7660079.png HTTP/1.1" 404 17 
2023-01-26 15:35:56,021 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/05/Lukashenko_potato_big.jpg HTTP/1.1" 404 17 
2023-01-26 15:35:56,021 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/07/15800683022690.jpg HTTP/1.1" 404 17 
2023-01-26 15:35:56,021 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/06/You_Shall_Not_Pass_0-1_screenshot.jpg HTTP/1.1" 404 17 
2023-01-26 15:35:56,022 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/03/iStock-579733578-e1573574017848-1024x499.jpg HTTP/1.1" 404 17 
2023-01-26 15:35:56,029 WARNING  log log.py Not Found: /media/photos/2023/01/02/scale_1200.webp 
2023-01-26 15:35:56,031 WARNING  log log.py Not Found: /media/photos/2022/12/28/84482761-e1af-4749-ac54-ac2d26e2e8fb.jpg 
2023-01-26 15:35:56,032 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/scale_1200.webp HTTP/1.1" 404 17 
2023-01-26 15:35:56,033 WARNING  log log.py Not Found: /media/photos/2023/01/02/slug-3612269_1.jpg 
2023-01-26 15:35:56,034 WARNING  basehttp basehttp.py "GET /media/photos/2022/12/28/84482761-e1af-4749-ac54-ac2d26e2e8fb.jpg HTTP/1.1" 404 17 
2023-01-26 15:35:56,036 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/slug-3612269_1.jpg HTTP/1.1" 404 17 
2023-01-26 15:36:32,089 ERROR  log log.py Internal Server Error: / 
Traceback (most recent call last):
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\core\handlers\base.py", line 204, in _get_response
    response = response.render()
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\template\response.py", line 105, in render
    self.content = self.rendered_content
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\template\response.py", line 83, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\template\base.py", line 170, in render
    return self._render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\test\utils.py", line 100, in instrumented_test_render
    return self.nodelist.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\template\loader_tags.py", line 127, in render
    compiled_parent = self.get_parent(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\template\loader_tags.py", line 124, in get_parent
    return self.find_template(parent, context)
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\template\loader_tags.py", line 104, in find_template
    template_name, skip=history,
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\template\engine.py", line 125, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\template\loaders\cached.py", line 58, in get_template
    template = super().get_template(template_name, skip)
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\template\loaders\base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\template\base.py", line 155, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\template\base.py", line 193, in compile_nodelist
    return parser.parse()
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\template\base.py", line 478, in parse
    raise self.error(token, e)
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\template\base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\template\loader_tags.py", line 216, in do_block
    nodelist = parser.parse(('endblock',))
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\template\base.py", line 478, in parse
    raise self.error(token, e)
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\template\base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\template\defaulttags.py", line 968, in do_if
    nodelist = parser.parse(('elif', 'else', 'endif'))
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\template\base.py", line 449, in parse
    raise self.error(token, e)
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\template\base.py", line 447, in parse
    filter_expression = self.compile_filter(token.contents)
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\template\base.py", line 563, in compile_filter
    return FilterExpression(token, self)
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\template\base.py", line 657, in __init__
    filter_func = parser.find_filter(filter_name)
  File "C:\Users\Tom\PycharmProjects\django_blog\blog_venv\lib\site-packages\django\template\base.py", line 569, in find_filter
    raise TemplateSyntaxError("Invalid filter: '%s'" % filter_name)
django.template.exceptions.TemplateSyntaxError: Invalid filter: 'hsddd'
2023-01-26 15:36:32,102 ERROR  basehttp basehttp.py "GET / HTTP/1.1" 500 145 
2023-01-26 15:36:32,257 WARNING  log log.py Not Found: /favicon.ico 
2023-01-26 15:36:32,258 WARNING  basehttp basehttp.py "GET /favicon.ico HTTP/1.1" 404 17 
2023-01-26 15:37:03,315 WARNING  log log.py Not Found: /media/photos/2023/01/18/15910177301560.png 
2023-01-26 15:37:03,318 WARNING  log log.py Not Found: /media/photos/2023/01/15/scale_1200.webp 
2023-01-26 15:37:03,320 WARNING  log log.py Not Found: /media/photos/2023/01/09/16293175060633.jpg 
2023-01-26 15:37:03,321 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/15/scale_1200.webp HTTP/1.1" 404 17 
2023-01-26 15:37:03,321 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/18/15910177301560.png HTTP/1.1" 404 17 
2023-01-26 15:37:03,324 WARNING  log log.py Not Found: /media/photos/2023/01/02/543-9a6d5dc5b946eb9a62d2f62422af011c.png 
2023-01-26 15:37:03,324 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/09/16293175060633.jpg HTTP/1.1" 404 17 
2023-01-26 15:37:03,327 WARNING  log log.py Not Found: /media/photos/2023/01/10/7660079.png 
2023-01-26 15:37:03,330 WARNING  log log.py Not Found: /media/photos/2023/01/07/15800683022690.jpg 
2023-01-26 15:37:03,330 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/543-9a6d5dc5b946eb9a62d2f62422af011c.png HTTP/1.1" 404 17 
2023-01-26 15:37:03,331 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/10/7660079.png HTTP/1.1" 404 17 
2023-01-26 15:37:03,332 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/07/15800683022690.jpg HTTP/1.1" 404 17 
2023-01-26 15:37:03,374 WARNING  log log.py Not Found: /media/photos/2023/01/05/Lukashenko_potato_big.jpg 
2023-01-26 15:37:03,376 WARNING  log log.py Not Found: /media/photos/2023/01/06/You_Shall_Not_Pass_0-1_screenshot.jpg 
2023-01-26 15:37:03,377 WARNING  log log.py Not Found: /media/photos/2023/01/03/iStock-579733578-e1573574017848-1024x499.jpg 
2023-01-26 15:37:03,378 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/06/You_Shall_Not_Pass_0-1_screenshot.jpg HTTP/1.1" 404 17 
2023-01-26 15:37:03,379 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/05/Lukashenko_potato_big.jpg HTTP/1.1" 404 17 
2023-01-26 15:37:03,380 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/03/iStock-579733578-e1573574017848-1024x499.jpg HTTP/1.1" 404 17 
2023-01-26 15:37:03,395 WARNING  log log.py Not Found: /media/photos/2023/01/02/scale_1200.webp 
2023-01-26 15:37:03,397 WARNING  log log.py Not Found: /media/photos/2022/12/28/84482761-e1af-4749-ac54-ac2d26e2e8fb.jpg 
2023-01-26 15:37:03,398 WARNING  log log.py Not Found: /media/photos/2023/01/02/slug-3612269_1.jpg 
2023-01-26 15:37:03,401 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/scale_1200.webp HTTP/1.1" 404 17 
2023-01-26 15:37:03,401 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/slug-3612269_1.jpg HTTP/1.1" 404 17 
2023-01-26 15:37:03,402 WARNING  basehttp basehttp.py "GET /media/photos/2022/12/28/84482761-e1af-4749-ac54-ac2d26e2e8fb.jpg HTTP/1.1" 404 17 
2023-01-26 15:37:12,369 WARNING  log log.py Not Found: /media/photos/2023/01/05/Lukashenko_potato_big.jpg 
2023-01-26 15:37:12,370 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/05/Lukashenko_potato_big.jpg HTTP/1.1" 404 17 
2023-01-26 15:37:13,109 WARNING  log log.py Not Found: /media/photos/2023/01/18/15910177301560.png 
2023-01-26 15:37:13,110 WARNING  log log.py Not Found: /media/photos/2023/01/15/scale_1200.webp 
2023-01-26 15:37:13,111 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/18/15910177301560.png HTTP/1.1" 404 17 
2023-01-26 15:37:13,111 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/15/scale_1200.webp HTTP/1.1" 404 17 
2023-01-26 15:37:13,173 WARNING  log log.py Not Found: /media/photos/2023/01/09/16293175060633.jpg 
2023-01-26 15:37:13,174 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/09/16293175060633.jpg HTTP/1.1" 404 17 
2023-01-26 19:27:17,269 ERROR  log log.py Internal Server Error: / 
Traceback (most recent call last):
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\core\handlers\base.py", line 204, in _get_response
    response = response.render()
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\response.py", line 105, in render
    self.content = self.rendered_content
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\response.py", line 83, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\base.py", line 170, in render
    return self._render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\test\utils.py", line 100, in instrumented_test_render
    return self.nodelist.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\loader_tags.py", line 150, in render
    return compiled_parent._render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\test\utils.py", line 100, in instrumented_test_render
    return self.nodelist.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\loader_tags.py", line 62, in render
    result = block.nodelist.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\defaulttags.py", line 214, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\defaulttags.py", line 449, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\urls\base.py", line 86, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\urls\resolvers.py", line 698, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'add_page' not found. 'add_page' is not a valid view function or pattern name.
2023-01-26 19:27:17,279 ERROR  basehttp basehttp.py "GET / HTTP/1.1" 500 145 
2023-01-26 19:27:17,524 WARNING  log log.py Not Found: /favicon.ico 
2023-01-26 19:27:17,526 WARNING  basehttp basehttp.py "GET /favicon.ico HTTP/1.1" 404 17 
2023-01-26 19:28:29,023 WARNING  log log.py Not Found: /media/photos/2023/01/18/15910177301560.png 
2023-01-26 19:28:29,025 WARNING  log log.py Not Found: /media/photos/2023/01/15/scale_1200.webp 
2023-01-26 19:28:29,027 WARNING  log log.py Not Found: /media/photos/2023/01/09/16293175060633.jpg 
2023-01-26 19:28:29,028 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/18/15910177301560.png HTTP/1.1" 404 17 
2023-01-26 19:28:29,029 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/15/scale_1200.webp HTTP/1.1" 404 17 
2023-01-26 19:28:29,031 WARNING  log log.py Not Found: /media/photos/2023/01/02/543-9a6d5dc5b946eb9a62d2f62422af011c.png 
2023-01-26 19:28:29,033 WARNING  log log.py Not Found: /media/photos/2023/01/10/7660079.png 
2023-01-26 19:28:29,034 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/09/16293175060633.jpg HTTP/1.1" 404 17 
2023-01-26 19:28:29,036 WARNING  log log.py Not Found: /media/photos/2023/01/05/Lukashenko_potato_big.jpg 
2023-01-26 19:28:29,038 WARNING  log log.py Not Found: /media/photos/2023/01/06/You_Shall_Not_Pass_0-1_screenshot.jpg 
2023-01-26 19:28:29,040 WARNING  log log.py Not Found: /media/photos/2023/01/07/15800683022690.jpg 
2023-01-26 19:28:29,042 WARNING  log log.py Not Found: /media/photos/2023/01/03/iStock-579733578-e1573574017848-1024x499.jpg 
2023-01-26 19:28:29,042 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/543-9a6d5dc5b946eb9a62d2f62422af011c.png HTTP/1.1" 404 17 
2023-01-26 19:28:29,043 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/10/7660079.png HTTP/1.1" 404 17 
2023-01-26 19:28:29,044 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/07/15800683022690.jpg HTTP/1.1" 404 17 
2023-01-26 19:28:29,044 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/05/Lukashenko_potato_big.jpg HTTP/1.1" 404 17 
2023-01-26 19:28:29,044 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/06/You_Shall_Not_Pass_0-1_screenshot.jpg HTTP/1.1" 404 17 
2023-01-26 19:28:29,046 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/03/iStock-579733578-e1573574017848-1024x499.jpg HTTP/1.1" 404 17 
2023-01-26 19:28:29,050 WARNING  log log.py Not Found: /media/photos/2023/01/02/scale_1200.webp 
2023-01-26 19:28:29,052 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/scale_1200.webp HTTP/1.1" 404 17 
2023-01-26 19:28:29,055 WARNING  log log.py Not Found: /media/photos/2022/12/28/84482761-e1af-4749-ac54-ac2d26e2e8fb.jpg 
2023-01-26 19:28:29,056 WARNING  log log.py Not Found: /media/photos/2023/01/02/slug-3612269_1.jpg 
2023-01-26 19:28:29,057 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/slug-3612269_1.jpg HTTP/1.1" 404 17 
2023-01-26 19:28:29,057 WARNING  basehttp basehttp.py "GET /media/photos/2022/12/28/84482761-e1af-4749-ac54-ac2d26e2e8fb.jpg HTTP/1.1" 404 17 
2023-01-26 19:30:58,588 ERROR  log log.py Internal Server Error: / 
Traceback (most recent call last):
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\core\handlers\base.py", line 204, in _get_response
    response = response.render()
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\response.py", line 105, in render
    self.content = self.rendered_content
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\response.py", line 83, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\base.py", line 170, in render
    return self._render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\test\utils.py", line 100, in instrumented_test_render
    return self.nodelist.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\loader_tags.py", line 150, in render
    return compiled_parent._render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\test\utils.py", line 100, in instrumented_test_render
    return self.nodelist.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\loader_tags.py", line 62, in render
    result = block.nodelist.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\defaulttags.py", line 214, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\defaulttags.py", line 449, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\urls\base.py", line 86, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\urls\resolvers.py", line 698, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'add_page' not found. 'add_page' is not a valid view function or pattern name.
2023-01-26 19:30:58,598 ERROR  basehttp basehttp.py "GET / HTTP/1.1" 500 145 
2023-01-26 19:33:34,882 WARNING  log log.py Not Found: /media/photos/2023/01/18/15910177301560.png 
2023-01-26 19:33:34,885 WARNING  log log.py Not Found: /media/photos/2023/01/15/scale_1200.webp 
2023-01-26 19:33:34,893 WARNING  log log.py Not Found: /media/photos/2023/01/09/16293175060633.jpg 
2023-01-26 19:33:34,894 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/18/15910177301560.png HTTP/1.1" 404 17 
2023-01-26 19:33:34,895 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/15/scale_1200.webp HTTP/1.1" 404 17 
2023-01-26 19:33:34,897 WARNING  log log.py Not Found: /media/photos/2023/01/02/543-9a6d5dc5b946eb9a62d2f62422af011c.png 
2023-01-26 19:33:34,898 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/09/16293175060633.jpg HTTP/1.1" 404 17 
2023-01-26 19:33:34,903 WARNING  log log.py Not Found: /media/photos/2023/01/10/7660079.png 
2023-01-26 19:33:34,905 WARNING  log log.py Not Found: /media/photos/2023/01/06/You_Shall_Not_Pass_0-1_screenshot.jpg 
2023-01-26 19:33:34,907 WARNING  log log.py Not Found: /media/photos/2023/01/07/15800683022690.jpg 
2023-01-26 19:33:34,908 WARNING  log log.py Not Found: /media/photos/2023/01/03/iStock-579733578-e1573574017848-1024x499.jpg 
2023-01-26 19:33:34,909 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/543-9a6d5dc5b946eb9a62d2f62422af011c.png HTTP/1.1" 404 17 
2023-01-26 19:33:34,913 WARNING  log log.py Not Found: /media/photos/2023/01/05/Lukashenko_potato_big.jpg 
2023-01-26 19:33:34,916 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/10/7660079.png HTTP/1.1" 404 17 
2023-01-26 19:33:34,917 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/06/You_Shall_Not_Pass_0-1_screenshot.jpg HTTP/1.1" 404 17 
2023-01-26 19:33:34,918 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/07/15800683022690.jpg HTTP/1.1" 404 17 
2023-01-26 19:33:34,919 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/03/iStock-579733578-e1573574017848-1024x499.jpg HTTP/1.1" 404 17 
2023-01-26 19:33:34,923 WARNING  log log.py Not Found: /media/photos/2023/01/02/scale_1200.webp 
2023-01-26 19:33:34,925 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/05/Lukashenko_potato_big.jpg HTTP/1.1" 404 17 
2023-01-26 19:33:34,928 WARNING  log log.py Not Found: /media/photos/2022/12/28/84482761-e1af-4749-ac54-ac2d26e2e8fb.jpg 
2023-01-26 19:33:34,930 WARNING  log log.py Not Found: /media/photos/2023/01/02/slug-3612269_1.jpg 
2023-01-26 19:33:34,934 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/scale_1200.webp HTTP/1.1" 404 17 
2023-01-26 19:33:34,935 WARNING  basehttp basehttp.py "GET /media/photos/2022/12/28/84482761-e1af-4749-ac54-ac2d26e2e8fb.jpg HTTP/1.1" 404 17 
2023-01-26 19:33:34,936 WARNING  basehttp basehttp.py "GET /media/photos/2023/01/02/slug-3612269_1.jpg HTTP/1.1" 404 17 
2023-01-26 23:03:15,751 ERROR  log log.py Internal Server Error: / 
Traceback (most recent call last):
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\core\handlers\base.py", line 204, in _get_response
    response = response.render()
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\response.py", line 105, in render
    self.content = self.rendered_content
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\response.py", line 83, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\base.py", line 170, in render
    return self._render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\test\utils.py", line 100, in instrumented_test_render
    return self.nodelist.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\loader_tags.py", line 150, in render
    return compiled_parent._render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\test\utils.py", line 100, in instrumented_test_render
    return self.nodelist.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\loader_tags.py", line 62, in render
    result = block.nodelist.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\base.py", line 938, in render
    bit = node.render_annotated(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\defaulttags.py", line 214, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\base.py", line 905, in render_annotated
    return self.render(context)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\template\defaulttags.py", line 449, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\urls\base.py", line 86, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\Tom\PycharmProjects\django_blog\python_blog\blog_venv\lib\site-packages\django\urls\resolvers.py", line 698, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'addpage' not found. 'addpage' is not a valid view function or pattern name.
2023-01-26 23:03:15,763 ERROR  basehttp basehttp.py "GET / HTTP/1.1" 500 145 
